runtime/permissions/prompter.rs

```diff
@@ -198,7 +205,11 @@ impl PermissionPrompter for TtyPrompter {
     let _stderr_guard = std::io::stderr().lock();
 
     // print to stderr so that if stdout is piped this is still displayed.
-    const OPTS: &str = "[y/n] (y = yes, allow; n = no, deny)";
+    let opts: &str = if is_unary {
+      "[y/n/Y/N] (y = yes, allow; n = no, deny; Y = yes to all, allow all; N = no to all, deny all)"
```

Comment:
Some feedback here: `y/Y` and `n/N` are very confusing. I suggest to use other letters for "allow all"/"deny all" - how about "a" and "d" respectively? I think that might also fix problem with the tests you're having

Reply:
Good idea. But I don't think it'll fix the tests. They should be failing right but aren't irrespective of this change.

Reply:
Done.